### PR TITLE
Add GitHub Actions

### PR DESCRIPTION
**Pull request by phoracek**

### CODE REVIEW

**Comment by phoracek** [comment]
@RamLavi it takes 20 minutes due to emulation mode, but tests then finish. BTW this PR also fixes an issue in the test script.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up dependencies
+        run: |
+          sudo add-apt-repository ppa:smoser/swtpm
+          sudo apt-get update
+          sudo apt-get install qemu qemu-system-x86 qemu-utils npm swtpm
```
Comment (RamLavi) [inline .github/workflows/ci.yml:22]: Let's use the [setup script](https://github.com/RHsyseng/rhcos-slb/blob/main/tests/setup.sh), so we don't have to update this workflow every time we need to update one of our dependencies

**Review by RamLavi — CHANGES_REQUESTED** [review]
Looks good overall! added one comment.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (phoracek) [reply .github/workflows/ci.yml:22]: We will then need to maintain two setup scripts, one for `dnf`, one for `apt`. Is that still desirable?

Comment (RamLavi) [reply .github/workflows/ci.yml:22]: keeping 2 setup scripts is only slightly better than this, cause we would still need to maintain both separately. 
Let's keep it like this, then, imo it's not worth it..

Comment (phoracek) [reply .github/workflows/ci.yml:22]: Ok ok, thanks. Could we merge this then?

Comment (RamLavi) [reply .github/workflows/ci.yml:22]: yep

**Review by RamLavi — APPROVED** [review]
/lgtm

**Pull request merged**